script.js

```diff
@@ -0,0 +1,95 @@
+const container = document.querySelector(".container");
+const resetBtn = document.querySelector(".reset");
+const gridSize = document.querySelector(".grid-size");
+const rainbowBtn = document.querySelector(".rainbow");
+const blackBtn = document.querySelector(".black");
+const eraseBtn = document.querySelector(".erase");
+let isBlack = true;
+
+rainbowBtn.addEventListener("click", () => {
+  let color = "rainbow";
+  setColor(color);
+});
+
+blackBtn.addEventListener("click", () => {
+  let color = "black";
+  setColor(color);
+});
+
+eraseBtn.addEventListener("click", () => {
+  let color = "erase";
+  setColor(color);
```

Comment:
How about you set the color to `rgba(255, 255, 246, 0.877)` here and get rid of the `else` branch on L83? That way the behavior of the _setColor_ btns become consistent.

With this, you could take it a step further and get rid of the `if/else` in setColor.